api/api.go

```diff
@@ -143,7 +143,7 @@ func NewAPIWithVersion(ctx context.Context, globalConfig *conf.GlobalConfigurati
 
 	corsHandler := cors.New(cors.Options{
 		AllowedMethods:   []string{http.MethodGet, http.MethodPost, http.MethodPut, http.MethodDelete},
-		AllowedHeaders:   []string{"Accept", "Authorization", "Content-Type", audHeaderName},
+		AllowedHeaders:   []string{"Accept", "Authorization", "Content-Type", audHeaderName, "X-Use-Cookie"},
```

Comment:
I think you can use the constant `useCookieHeader` instead here.

Reply:
Missed that, :ty: